packages/owl-bot/src/copy-code.ts

```diff
@@ -594,45 +576,60 @@ export async function regeneratePullRequest(
   }
   const pull = pulls.data[0];
 
+  // If something goes wrong while examining the state of the open pull request,
+  // report the error on the log and in a comment on the pull request.
+  const reportError = async (error: string) => {
+    logger.error(`Error in ${pull.html_url}:\n${error}`);
+    await octokit.pulls.createReviewComment({
```

Comment:
I used createIssueComment for similar purpose. Do you prefer a review comment?

Reply:
In this case, it's an open review that we need to comment on, so `createReviewComment()` seemed like the natural choice.